authorizer-app-backend/src/main/java/org/radarbase/authorizer/resources/RestSourceUserResource.kt

```diff
@@ -144,6 +143,23 @@ class RestSourceUserResource(
         return Response.noContent().header("user-removed", userId).build()
     }
 
+    @POST
+    @Path("revoke/sourceType/{sourceType}/externalId/{externalId}")
```

Comment:
Alternative to match other api: `DELETE source-clients/{type}/authorization/{serviceUserId}` and move it to the source-clients resource. It might be useful to add a get method there as well, to get the user info and perhaps the token of a given serviceUserId.